iron-menu-behavior.html

```diff
@@ -325,8 +334,13 @@
      */
     _onKeydown: function(event) {
       if (!this.keyboardEventMatchesKeys(event, 'up down esc')) {
+        this.cancelDebouncer('_clearSearchText');
+        
         // all other keys focus the menu item starting with that character
-        this._focusWithKeyboardEvent(event);
+        this._searchText += String.fromCharCode(event.keyCode).toLowerCase();
```

Comment:
I think `key` is the best here, but probably not supported as widely as you'd like.  `key` would allow typing things like `é` to be easily appended to `_searchText` AFAICT, whereas I'm not sure how well `keyCode` works for this (and it's also deprecated).